quill/api/src/entity.rs

```diff
@@ -92,16 +93,16 @@ impl Entity {
     ///
     /// The message sends as a "system" message.
     /// See [the wiki](https://wiki.vg/Chat) for more details.
-    pub fn send_message(&self, message: impl AsRef<str>) {
-        let message = message.as_ref();
+    pub fn send_message(&self, message: impl Into<Text>) {
+        let message = message.into().to_string();
```

Comment:
We should define some lint which either prefers `to_string()` or `to_owned()`.

Reply:
It seems that clippy already lints `"some_string_literal".to_string()` and suggests `to_owned()`.

Reply:
`.into()` returns `Text` and not `&str`. `to_string()` is derived from `Text`'s `Display` trait implementation, which serializes the text component with `serde_json` and returns the json string.